src/themes/minechat.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
+
+html,
+body {
+  overflow: hidden;
+  height: 100vh;
+  margin-top: 0;
+}
+
+body {
+  font-family: "Press Start 2P", cursive;
+  font-weight: 700;
+  font-size: 25px;
+}
+
+[data-twitch-chat] {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  overflow: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+[data-twitch-chat] > * {
+  margin-top: 0.6rem;
+}
+
+[data-twitch-chat] > *:last-child {
+  margin-bottom: 2rem;
+}
+
+[data-twitch-chat] [data-twitch-message] [data-twitch-message-reply] {
+  display: none;
+}
+
+/* css to add "operator crown" for the broadcaster */
+[data-twitch-sender-roles="broadcaster"] > .twitch-chat-sender::before {
+  --size: 1.5rem;
+  content: "";
+  display: inline-block;
+  background-color: var(--twitch-sender-color, var(--twitch-purple));
+  height: var(--size);
+  width: var(--size);
+  margin-inline-end: 1rem;
+  clip-path: polygon(50% 0%, 76% 26%, 100% 0, 100% 95%, 0 95%, 0 0, 24% 24%);
+}
+
+[data-twitch-sender-roles="broadcaster"] > .twitch-chat-sender::after {
```

Comment:
Same as above - it'd be safest to use `~=` when checking that the sender is a broadcaster.

Reply:
This is a copy paste error, I should have been to `data-twitch-message`